README.md

```diff
@@ -116,6 +116,11 @@ Each job has a job ID, with the command "jobs cancel" a running (or queued) job
 
 `testengine jobs status <job ID>`
 
+### Print an execution report to the console'
```

Comment:
Nice!

Reply:
Now this comment needs to be updated with infor about running the job synchronous.